### ISSUE TITLE
Add additional option(s) to edit tabs in "content of main page" - Android TV

### ISSUE THREAD

**Issue by johnth3doe** [TeamNewPipe/NewPipe#7022]
### Checklist

- [x] I checked, but didn't find any direct duplicates (open OR closed) of this issue in the repo.  
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
I would, as the title suggests, like for a three-dot (aka 'burger menu') to be added to the "content of main page" because on android TV, the only options are:
1) The top-right icon to "restore default"
2) The "+" sign to add another tab and 
3) The two horizontal lines ("=") that allow touchscreen devices to re-order the tabs.

I would alternatively suggest: 
1) An option to reorder the tabs by small "^ or v" icons at the far-right (where the "=" is) that would allow non-touch devices to reorder the tabs, and an "x" to remove the tab; or 
2) An edit mode that allows the user to have a tick-box (like the " - [x] " when filling in the template checklist correctly) that then allows to use the remote to move the highlighted box to the desired position. That being said, "click-and-hold" is not a viable option, in my humble opinion.

#### Is your feature request related to a problem? Please describe it
Some android box devices do not come with a remote that has the ability to "click-and-drag" or with a number-pad or microphone options, therefore some menus or options are not possible to interact with as intended (this request showing an example).

#### Additional context
If I am not mistaken, users cannot add a tag themselves, so I request that the "android tv" and/or any other relevant tag be added [at the discretion of the honorable NewPipe Team and contributors] and would like to express my gratitude to the continued dedication to this project and community in these trying times.

#### How will you/everyone benefit from this feature?
It would allow more users to take advantage of all the features that newpipe can offer, irrespective of mobile device, or as in this case, TV.

___
Thank you for your time and consideration.

Take care, be well.

**Comment by thebe14** [comment]
I agree with the author of this issue (was about to create a similar issue). Swipe cannot the method to remove, and drag and drop cannot be the method to reorder when running on a TV )or on any device that is not a phone, really). Please add traditional methods to do these actions, and make the swipe the backup one.
